docs/formula_grader.md

```diff
@@ -221,14 +162,35 @@ grader = FormulaGrader(
 )
 ```
 
-Constants are like variables that only ever have one value. You can overwrite constants if you like by passing in new definitions in user_constants, but we recommend against this. We also recommend ensuring that your constants and variables all have unique names. Variables will overwrite any constants that have the same name.
+Constants are like variables that only ever have one value.
 
 Note that by default, edX defines the constants `k`, `c` and `T`. In our experience, these constants lead to confusion when answers that should produce invalid variable errors instead simply eat a student's attempt. You can reintroduce them as necessary.
```

Comment:
These are now gone!!!!! I think we can remove this paragraph.

Reply:
You mean edX removed `k`, `c`, and `T`? All of the constants, or just those? Were those the only ones?

Reply:
Yup, they removed `k`, `c`, `Q` and `T` after I poked them :-) https://github.com/edx/edx-platform/pull/17368